addons/relate/ftests/src/main/java/org/commonjava/indy/relate/ftest/PomDownloadListenerTest.java

```diff
@@ -0,0 +1,68 @@
+package org.commonjava.indy.relate.ftest;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringUtils;
+import org.commonjava.indy.ftest.core.AbstractIndyFunctionalTest;
+import org.commonjava.indy.model.core.RemoteRepository;
+
+import java.io.InputStream;
+
+import org.commonjava.test.http.expect.ExpectationServer;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+
+import static org.commonjava.indy.model.core.StoreType.remote;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Created by ruhan on 2/17/17.
+ */
+public class PomDownloadListenerTest
+                extends AbstractIndyFunctionalTest
+{
+    private static final String path = "org/foo/bar/1/bar-1.pom";
+
+    private static final String pathRel = path + ".rel";
+
+    private static final String content =
+                    "<project><modelVersion>4.0.0</modelVersion><groupId>org.foo</groupId><artifactId>bar</artifactId><version>1</version></project>";
+
+    @Rule
+    public ExpectationServer server = new ExpectationServer();
+
+    @Test
+    public void run() throws Exception
+    {
+        final String repo1 = "repo1";
+
+        server.expect( server.formatUrl( repo1, path ), 200, content );
+
+        RemoteRepository remote1 = new RemoteRepository( repo1, server.formatUrl( repo1 ) );
+        client.stores().create( remote1, "adding remote", RemoteRepository.class );
+
+        InputStream is = client.content().get( remote, repo1, path );
+        String s = IOUtils.toString( is );
+        assertThat( s, equalTo( content ) );
+
+        // check .rel exists, timeout after 10 retries
```

Comment:
You might be able to use AbstractIndyFunctionalTest.waitForEventPropagation() here, instead of polling...I'm not sure.